src/_pytest/capture.py

```diff
@@ -117,7 +117,9 @@ def stop_global_capturing(self):
             self._global_capturing = None
 
     def resume_global_capture(self):
-        self._global_capturing.resume_capturing()
+        cap = getattr(self, "_global_capturing", None)
```

Comment:
the `getattr` here should be unnecessary -- it's only `None` while python tears down the process (and not a missing attribute)

I'd also leave a comment here along the lines of:

```python
    # during teardown of the python process, attributes are `None`d
    if self._global_capturing is not None:
        self._global_capturing.resume_capturing()
```